iphone/Classes/TiUITextWidget.m

```diff
@@ -160,6 +160,70 @@ -(void)setAutocorrect_:(id)value
 	[[self textWidgetView] setAutocorrectionType:[TiUtils boolValue:value] ? UITextAutocorrectionTypeYes : UITextAutocorrectionTypeNo];
 }
 
+-(void)setTextContentType_:(id)value
+{
+    ENSURE_TYPE_OR_NIL(value, NSString);
+    if (@available(iOS 10.0, *)) {
```

Comment:
We can not convert NSstring to __NSCFConstantString. If we do-
NSString *str = @"test string", it get automatically converted. It is done at compile time. Either we can simply write -
textContentType = @"username" 
or we write 
textContentType = UITextContentTypeUsername
In both cases we have to use if else condition for differentiation. First option is not good as we have to get the string value of each UITextContentType and write there.